pkg/webhook/cspc.go

```diff
@@ -285,17 +362,11 @@ func (poolValidator *PoolValidator) raidGroupValidation(
 	raidGroup *cstor.RaidGroup, rgType string) (bool, string) {
 
 	if len(raidGroup.BlockDevices) == 0 {
-		return false, fmt.Sprintf("number of block devices honouring raid type should be specified")
```

Comment:
Below check is good enough we don't require this check(We can remove in next PR).